userspace/Build.mk

```diff
@@ -283,9 +290,16 @@ build/userspace/$(APP)/$(BOARD)/full_image: \
 	arm-none-eabi-objcopy --update-section \
 		.apps=build/userspace/$(APP)/$(BOARD)/app.tbf \
 		build/userspace/$(APP)/$(BOARD)/unsigned_image
-	$(TANGO_CODESIGNER) --b --input build/userspace/$(APP)/$(BOARD)/unsigned_image \
-		--key=$(TANGO_CODESIGNER_KEY) \
-		--output=build/userspace/$(APP)/$(BOARD)/signed_image
+	if [ -n "${TANGO_CODESIGNER}" -a -n "${TANGO_CODESIGNER_KEY}" ]; then \
```

Comment:
The copy&paste is obviously not desirable, but you're not introducing it, so I think it's ok.

Reply:
Yeah -- three copies of almost identical code here.  Pretty clear how to abstract them into a single `define` -- maybe I'll do that after.

Reply:
I've worked up a change that abstracts those three copies into a single macro.  I will submit it after as a separate PR.

Reply:
See branch `factor-FULL_IMAGE_TARGET` of https://github.com/dann21/tock-on-titan.git.  I will cut a PR for what's on this branch once this PR has been disposed of.